products/servicemanagement/api.yaml

```diff
@@ -13,7 +13,7 @@
 
 --- !ruby/object:Api::Product
 name: ServiceManagement
-display_name: Service Management
+display_name: Cloud Endpoints
```

Comment:
I'm assuming this is intentional, but just in case, wanted to double-check.

Reply:
It is, thanks for checking! This API generates the `google_endpoints_service_iam_*` resources, so it  felt like a better name.